e2e/suites/metadataprops.test.ts

```diff
@@ -0,0 +1,166 @@
+      // eslint-disable-next-line no-await-in-loop
+      await propertyAddButtons[propIndex].click();
+    }
+
+    const propertiesInMetadata = await page.$$('.metadataTemplate li');
+    await expect(propertiesInMetadata[7]).toClick('button', { text: 'Edit' });
+    await expect(propertiesInMetadata[7]).toSelect('select:first-of-type', 'Relacionado a');
+    await expect(page).toClick('button', { text: 'Save' });
+    await expect(page).toClick('div.alert-success');
+  });
+
+  it('should create an entity filling all the props.', async () => {
+    await expect(page).toClick('a', { text: 'Private documents' });
+    await expect(page).toClick('span', { text: 'New entity' });
+    await expect(page).toFill(
+      'textarea[name="uploads.sidepanel.metadata.title"]',
+      'Entity with all props'
+    );
+    await expect(page).toSelect('select:first-of-type', 'All props');
+
+    await expect(page).toFill('.form-group.text input', 'demo text');
+    await expect(page).toFill('.form-group.numeric input', '42');
+    await expect(page).toSelect('.form-group.select select', 'Activo');
+    await scrollTo('.form-group.multiselect li.multiselectItem');
+    await expect(page).toClick('.form-group.multiselect li.multiselectItem', {
+      text: 'Activo',
+    });
+    await expect(page).toClick('.form-group.relationship li.multiselectItem', {
+      text: '19 Comerciantes',
+    });
+    await scrollTo('.form-group.date input');
+    await selectDate('.form-group.date input', '08/09/1966');
+    await selectDate('.form-group.daterange div.DatePicker__From input', '23/11/1963');
+    await selectDate('.form-group.daterange div.DatePicker__To input', '12/09/1964');
+    await expect(page).toClick('.form-group.multidate button.btn.add');
+    await selectDate('.form-group.multidate .multidate-item:first-of-type input', '23/11/1963');
+    await selectDate('.form-group.multidate .multidate-item:nth-of-type(2) input', '12/09/1964');
+    await expect(page).toClick('.form-group.multidaterange button.btn.add');
+    await selectDate(
+      '.form-group.multidaterange .multidate-item:first-of-type div.DatePicker__From input',
+      '23/11/1963'
+    );
+    await selectDate(
+      '.form-group.multidaterange .multidate-item:first-of-type div.DatePicker__To input',
+      '12/09/1964'
+    );
+    await selectDate(
+      '.form-group.multidaterange .multidate-item:nth-of-type(2) div.DatePicker__From input',
+      '23/11/1963'
+    );
+    await selectDate(
+      '.form-group.multidaterange .multidate-item:nth-of-type(2) div.DatePicker__To input',
+      '12/09/1964'
+    );
+    await expect(page).toFill('.form-group.markdown textarea', '***smile***');
+    await expect(page).toFill('.form-group.link #label', 'Huridocs');
+    await expect(page).toFill('.form-group.link #url', 'https://www.huridocs.org/');
+
+    await expect(page).toClick('button', { text: 'Save' });
+    await expect(page).toClick('div.alert-success');
+  }, 60000);
```

Comment:
Locally it works without the timeouts, is it failing on GH?

Reply:
This is something that was used in the previous implementation. I'm not sure about the reason, since I see now actions there that might hang. Probably you are right and we can remove this.